adr/0008-code-owners.md

```diff
@@ -0,0 +1,100 @@
+# 8. Code owners
+
+Date: 2019-09-18
+
+Architecture issue: [#287](https://github.com/home-assistant/architecture/issues/287)
+
+## Status
+
+Draft
+
+## Context
+
+We get contributed a lot of new integrations, new features to integrations and refactors of integrations. The Home Assistant project is honored to receive so many great contributions to our project!
+
+Unfortunately, as a contributor, adding oneself as (the, or one of the) code owners of the integration contributed or contributed to, doesn't happen spontaneously that often.
+
+Not adding oneself as a code owner has drawbacks for the project:
+
+- The contributor doesn't "own" (in terms of taking responsibility) his code, and thus contribution, in a more formal fashion.
+- Without being listed as a code owner, our GitHub bot will not notify the contributor, when an issue for the integration is reported, quite possibly affecting his contribution.
+- Most integrations end up with having no or a single code owner.
```

Comment:
The low percent codeowners is the result of our history and not how things are moving forward at the moment.

Reply:
I agree, however, this ADR also covers the recruitment of new code owners for those older integrations (any for that matter). Hence, still true. However, I see your point, and will rephrase it a bit to match your comment.

Reply:
Rephrased this part to not draw conclusions, but still, indicate the risk.
Please let me know if this is better @MartinHjelmare.

Reply:
Good!